doc/README.md

```
@@ -90,3 +91,4 @@ Each package name consist from three parts
 [Build a Reliable Package Source]: ./ReliablePackageSource.md
 [Build Process]:                   ./BuildProcess.md
 [Package Dependencies]:            ./PackageDependencies.md
+[Git Repository]:                  ./UseCaseScenarios.md
```

Markdownlint (0.37.0)

Unused link or image reference definition: "git repository"
Link and image reference definitions should be needed

(MD053, link-image-reference-definitions)